.github/workflows/benchmark-pull-request.yml

```diff
@@ -0,0 +1,145 @@
+name: Run performance benchmark on pull request
+on:
+  issue_comment:
```

Comment:
This part confuses me: the benchmark run is on pull request (make sense) but comment is on the issue (confusion)? I would expect it to be on the pull request, where the code lives. Plus, the issue could have several pull requests "associated" with it, fe multiple  backports or large features split in chunks.